File: exercice4/main.ts
class Customer {
    private name: string;
    private bankAccountId: number;

    constructor(name: string, bankAccountId: number) {
        this.name = name;
        this.bankAccountId = bankAccountId;
    }

    getName(): string {
        return this.name;
    }

    getBankAccountId(): number {
        return this.bankAccountId;
    }

    public setName(name: string): void {
        this.name = name;
    }
}

let toupiBankAccountInfo: Customer = new Customer("Toupi", 14);
console.log(toupiBankAccountInfo.getBankAccountId());

toupiBankAccountInfo.setName("Nep");
console.log(toupiBankAccountInfo.getName());

File: exercice4/main.java
class Playground {
    public static void main(String[ ] args) {
        Customer toupi = new Customer("Toupi", 19);
        System.out.println(toupi.getBankId());
    }
}

class Customer {
    private String name;
    private int bankId;


    public Customer(String name, int bankId) {
        this.name = name;
        this.bankId = bankId;
    }

    public int getBankId() {
        return this.bankId;
    }
}
